bash_command_line_101/bash_command_line_101.md

```diff
@@ -207,6 +222,18 @@ Pictures
 Public
 ```
 
```

Comment:
@pm0kjp make a suggestion here to move this north of the first use of "+ output" and maybe enclose it in a highlight box.